docs/dev/recipe-client-addon/workflow.rst

```diff
@@ -71,6 +71,9 @@ files:
    cd recipe-client-addon
    ./bin/make-xpi.sh # Creates /recipe-client-addon/shield-recipe-client.xpi
 
+Pass the ``-b``/``--build-vendor`` flag to auto-build vendor dependencies before
+creating the XPI.
```

Comment:
I think that the default value here should be to build the vendor library, or to autodetect if it hasn't ever been built. I'd be ok if we didn't rebuild by default. This is mostly for ease of  use for new or infrequent contributors.